qemu_amd64.go

```diff
@@ -0,0 +1,244 @@
+//
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// +build !arm64
+
+package virtcontainers
+
+import (
+	"fmt"
+	"os"
+
+	govmmQemu "github.com/intel/govmm/qemu"
+)
+
+type qemuAmd64 struct {
+	// inherit from qemuKVM, overwrite methods if needed
+	qemuKVM
+}
+
+const defaultQemuPath = "/usr/bin/qemu-system-x86_64"
+
+const defaultQemuMachineType = "pc"
+
+const defaultQemuMachineAccelerators = "kvm,kernel_irqchip,nvdimm"
+
+const (
+	// QemuPCLite is the QEMU pc-lite machine type
+	QemuPCLite = "pc-lite"
+
+	// QemuPC is the QEMU pc machine type
+	QemuPC = defaultQemuMachineType
+
+	// QemuQ35 is the QEMU Q35 machine type
+	QemuQ35 = "q35"
+)
+
+var qemuPaths = map[string]string{
+	QemuPCLite: "/usr/bin/qemu-lite-system-x86_64",
+	QemuPC:     defaultQemuPath,
+	QemuQ35:    defaultQemuPath,
+}
+
+// kernelParamsNonDebug is a list of the default kernel
+// parameters that will be used in standard (non-debug) mode.
+var kernelParamsNonDebug = []Param{
```

Comment:
These params (and `kernelParamsDebug`) are common for all supported architectures so maybe could be moved to `qemu_kvm.go` or even `qemu.go`?